osutil/user.go

```diff
@@ -161,3 +157,21 @@ func RealUser() (*user.User, error) {
 
 	return real, nil
 }
+
+// UidGid returns the uid and gid of the given user, as uint32s
+//
+// XXX this should go away soon
+func UidGid(u *user.User) (uint32, uint32, error) {
+	// XXX this will be wrong for high uids on 32-bit arches (for now)
+	const flagID = uint32(1<<32 - 1)
```

Comment:
Can you use `osutil/sys.FlagID` here or is there some cyclic import?

Reply:
in the code i see i do. not sure when you reviewed :-)